DATS.json

```diff
@@ -8,24 +8,23 @@
 	],
     "creators": [
         {
-            "@type": "Organization",
             "name": "BigBrain project"
         }
     ],
     "types": [
         {
 		"information":
-		{
+			{
 	            "value": "histology"
-		}
+			}
         }
     ],
     "distributions": [
 	        {
 			"formats": [
 				"MINC"
```

Comment:
In the list of formats available, NIfTI should also be mentioned for this dataset